### PR TITLE
Update docs structure 1

### PR DESCRIPTION
**Pull request by olivekl**
First in a series of PRs to update docs; see [Issue 672](https://github.com/slsa-framework/slsa/issues/672)

- Move content and create pages according to [Issue 672](https://github.com/slsa-framework/slsa/issues/672)
- No changes to the moved content (content updates are forthcoming in another PR)
- Add descriptive material to pages/sections for clarity about what’s moved
- Add or update Previous/Next nav buttons
Note: some nav buttons are missing because not all files are in the Spec directory yet (will move and add in later PR)

### CODE REVIEW

File: docs/spec/v1.0/levels.md [hunk 1]
```diff
@@ -17,60 +17,9 @@ tampered with and can be securely traced back to its source.
...
-
--   **Organizations,** who are both producers and consumers of software. In
-    addition to the goals above, organizations also want to broadly understand
-    and reduce risk across the organization.
-
--   **Infrastructure providers,** such as build services and package
-    ecosystems, who are critical to achieving SLSA. While not the primary
-    beneficiary of SLSA, they may benefit from increased security and user
-    trust.
+This page is a descriptiove overview of the SLSA levels and tracks, describing their intent. For the prescriptive requirements for each level, see
```
Comment (kpk47) [inline docs/spec/v1.0/levels.md:20]:
```suggestion
This page is a descriptive overview of the SLSA levels and tracks, describing their intent. For the prescriptive requirements for each level, see
```

**Review by kpk47 — APPROVED** [review]

File: docs/spec/v1.0/levels.md (hunk 1 above)
Comment (olivekl) [reply docs/spec/v1.0/levels.md]: done, thanks!

File: docs/spec/v1.0/levels.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+  url: terminology
 next_page:
-  title: Guiding principles
-  url: principles
+  title: Producing artifacts
+  url: requirements
 ---
 
-# Security levels
+# Security Levels
```
Comment (MarkLodato) [inline docs/spec/v1.0/levels.md:10]: nit: seems like capitalization is inconsistent. For example, this one was changed to title case, while index and whats-new were changed to sentence case.

File: docs/use-cases.md [hunk 3]
```diff
@@ -1,5 +1,12 @@
 ---
 title: Use cases
+prev_page:
+  title: Supply-chain threats TODO
+  url: 
+next_page:
+  title: FAQ TODO
+  url: 
+
```
Comment (MarkLodato) [inline docs/use-cases.md:9]: revert this change (wait until we move the use-cases page)

File: docs/spec/v1.0/threats.md [hunk 4]
```diff
@@ -1,129 +1,18 @@
 ---
-title: Threats and mitigations
 prev_page:
-  title: Verifying Build Systems
+  title: Verifying build systems
   url: verifying-systems
 next_page:
-  title: Frequently Asked Questions
+  title: FAQ
   url: faq
```
Comment (MarkLodato) [inline docs/spec/v1.0/threats.md:7]: Remove - this is the last page now

File: docs/spec/v1.0/threats.md [hunk 5]
```diff
@@ -1,129 +1,18 @@
 ---
-title: Threats and mitigations
 prev_page:
-  title: Verifying Build Systems
+  title: Verifying build systems
   url: verifying-systems
```
Comment (MarkLodato) [inline docs/spec/v1.0/threats.md:4]: should be verifying artifacts

File: docs/spec/v1.0/terminology.md [hunk 6]
```diff
@@ -1,20 +1,16 @@
 ---
-title: Terminology
 prev_page:
-  title: Guiding principles
-  url: principles
+  title: Future directions
+  url: future-directions
```
Comment (MarkLodato) [inline docs/spec/v1.0/terminology.md:4]: Need to update future-directions.md to have terminology as the next_page

File: docs/spec/v1.0/threats-overview.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+  url: principles
+next_page:
+  title: FAQ
+  url: faq
+---
+# Supply-chain threats 
+
+Attacks can occur at every link in a typical software supply chain, and these
+kinds of attacks are increasingly public, disruptive, and costly in today's
+environment. [TODO: add link to Threats in detail here and bottom]
```
Comment (MarkLodato) [inline docs/spec/v1.0/threats-overview.md:13]: did you mean to do this now?

**Review by MarkLodato — CHANGES_REQUESTED** [review]

File: docs/spec/v1.0/index.md [hunk 8]
```diff
@@ -36,17 +36,51 @@ Known issues:
...
+## Attestation formats
+
+These pages include the concrete schemas for SLSA attestations. The Provenance
+and VSA formats are recommended, but not required by the specification.
+
+| Page | Description |
+| ---- | --- |
+| [General model](/attestation-model) | General attestation model|
+| [Provenance](/provenance/v1) | Suggested provenance format and explanation |
+| [VSA](/verification_summary/v1) | Suggested VSA format and explanation |
```
Comment (MarkLodato) [inline docs/spec/v1.0/index.md:75]:
```suggestion
| [Provenance](/provenance/v1) | Recommended format for provenance |
| [VSA](/verification_summary/v1) | Recommended format for summarizing verification results |
```

File: docs/spec/v1.0/levels.md (hunk 2 above)
Comment (olivekl) [reply docs/spec/v1.0/levels.md:10]: thanks for catching -- aiming for sentence case even for titles, right?

**Review by MarkLodato — APPROVED** [review]

**Pull request merged**